logs/parse.go

```diff
@@ -541,13 +506,77 @@ func ParseLogLineWithPrefix(prefix string, line string) (logLine state.LogLine,
 	return
 }
 
+func getOccurredAt(timePart string, tz *time.Location, rsyslog bool) time.Time {
+	if tz != nil && !rsyslog {
+		lastSpaceIdx := strings.LastIndex(timePart, " ")
+		if lastSpaceIdx == -1 {
+			return time.Time{}
+		}
+		timePartNoTz := timePart[0:lastSpaceIdx]
+		result, err := time.ParseInLocation("2006-01-02 15:04:05", timePartNoTz, tz)
+		if err != nil {
+			return time.Time{}
```

Comment:
I originally fell back to the existing parsing if this fails, but I feel like failure here is unlikely and we can just consider the line unparsable, like we were doing for other time parsing errors before.